src/EditorFeatures/Core/Implementation/InlineRename/InlineRenameSession.OpenTextBufferManager.cs

```diff
@@ -245,10 +246,25 @@ private void OnTextBufferChanged(object sender, TextContentChangedEventArgs args
                 }
             }
 
+            /// <summary>
+            /// This is a work around for a bug in Razor where the projection spans can get out-of-sync with the
+            /// identifiers.  When that bug is fixed this helper can be deleted.
+            /// </summary>
```

Comment:
We're still trying to track down whether or not the bug is in Razor's handling of the projection span or if it's an editor bug.  Once we have a concrete work item I'll reference it here with the appropriate URL.